src/include/daos/common.h

```diff
@@ -558,7 +558,8 @@ daos_unparse_ctype(daos_cont_layout_t ctype, char *string)
 		strcpy(string, "HDF5");
 		break;
 	default:
-		D_ASSERT(0);
+		strcpy(string, "unknown");
```

Comment:
let's make it all caps

Reply:
Should that be done also for the other function daos_unparse_oclass()? And for the tiny, small, large and repl_max object classes in addition to "unknown" -> "UNKNOWN"